subgraph/src/fetch/account.ts

```diff
@@ -1,13 +1,9 @@
-import {
-    Address,
-} from '@graphprotocol/graph-ts'
+import { Address, BigInt } from "@graphprotocol/graph-ts";
 
-import {
-    Account,
-} from '../../generated/schema'
+import { Account } from "../../generated/schema";
 
-export function fetchAccount(address: Address) : Account {
-    let account = new Account(address.toHex())
-    account.save()
-    return account
-}
\ No newline at end of file
+export function fetchAccount(address: string): Account {
```

Comment:
Any reason to do that ?

If anything, I believe the change should be toward using `id: Bytes!` for the account entity